webui/react/src/components/kit/Card.tsx

```diff
@@ -0,0 +1,90 @@
+import { Dropdown, MenuProps, Space } from 'antd';
+import React from 'react';
+
+import { ConditionalWrapper } from 'components/ConditionalWrapper';
+import Link from 'components/Link';
+import Icon from 'shared/components/Icon';
+
+import Button from './Button';
+import css from './Card.module.scss';
+
+interface CardProps {
+  actionMenu?: MenuProps;
+  children?: React.ReactNode;
+  clickable?: boolean;
+  disabled?: boolean;
+  height?: number;
+  href?: string;
+  onClick?: () => void;
+  width?: number;
+}
+
+const stopPropagation = (e: React.MouseEvent): void => e.stopPropagation();
+
+const Card: Card = ({
+  actionMenu,
+  children,
+  clickable,
+  disabled = false,
+  onClick,
+  height = 184,
+  width = 184,
+  href,
```

Comment:
Should `onClick` and `href` be exclusive?

Reply:
Yes, I can't imagine a reason why we would want both. Is there a good way to enforce that?

Reply:
You could do something like `onClick?: string | () => void;` and then handle the string case as an href. If you wanted to make it more explicit you could introduce an ADT:
```typescript
<Card onClick={Card.LinkTo("mycoollink")} />
```

Reply:
Oh also there's another option here which is to make the args themselves a sum type:
```typescript
type CardPropsBase = { ... }
type CardProps = ({ href: string } & CardPropsBase) | ({ onClick: () => void } & CardPropsBase)
```